src/uct/api/uct.h

```diff
@@ -251,6 +251,10 @@ typedef enum uct_atomic_op {
 #define UCT_IFACE_FLAG_EVENT_RECV_SIG  UCS_BIT(48) /**< Event notification of signaled tag and active
                                                         message is supported */
 
+#define UCT_IFACE_FLAG_SUPPORT_PRESENT UCS_BIT(49) /**< Supported by every device version.
```

Comment:
thinking of it again, i don't like this flag
instead, let's put in UCP a configuration of which TLs to pack as part of the partial address (similar to memory allocation config, for example). It will currently be "ud,ud_x"

Reply:
you mean introduce an env paramter for TLs to pack as part of the partial address?
like https://github.com/openucx/ucx/blob/master/src/ucp/core/ucp_context.h#L167 ?
I think it would add complexity to the code. Isn't having a flag simpler in this case?
Do we want to allow the user to control this logic?